google/cloud/spanner/client.h

```diff
@@ -717,14 +716,9 @@ class Client {
   StatusOr<PartitionedDmlResult> ExecutePartitionedDml(SqlStatement statement,
                                                        Options opts = {});
 
-  ///@{
-  /// @name Backwards compatibility for ClientOptions.
-  explicit Client(std::shared_ptr<Connection> conn, ClientOptions const& opts)
-      : Client(std::move(conn), Options(opts)) {}
   explicit Client(std::shared_ptr<Connection> conn,
                   std::initializer_list<internal::NonConstructible>)
       : Client(std::move(conn)) {}
```

Comment:
This constructor is part of the `ClientOptions` backwards compatibility, and should be removed too.